model/camera/utils.go

```diff
@@ -19,21 +20,34 @@ func getHomeDir() (string, error) {
 	return home, nil
 }
 
-// ToSaveCommand converts a do command to a *videostore.SaveRequest.
-func ToSaveCommand(command map[string]interface{}) (*videostore.SaveRequest, error) {
+// parseTimeRange parses the from/to timestamps from a command.
+func parseTimeRange(command map[string]interface{}) (from, to time.Time, err error) {
 	fromStr, ok := command["from"].(string)
 	if !ok {
-		return nil, errors.New("from timestamp not found")
+		return time.Time{}, time.Time{}, errors.New("from timestamp not found")
 	}
-	from, err := videostore.ParseDateTimeString(fromStr)
+	from, err = videostore.ParseDateTimeString(fromStr)
```

Comment:
Just to clarify we are still assuming that the caller of fetch/save are providing from/to date-times in the video-store machine's local timezone?

Do we have an upgrade path to eventually allow users format queries in UTC to avoid the complication of having to know target machines timezone?

Reply:
I can make a ticket rn

Reply:
https://viam.atlassian.net/browse/RSDK-10317